packages/doc/content/components/components/chips/chips-web.mdx

```diff
@@ -0,0 +1,29 @@
+### Reference
+
+Gympass `<Chips />` description
+
+### Usage
+
+```javascript state
+render(() => {
+  const [first, toggleFirst] = useState(false);
+  const [second, toggleSecond] = useState(false);
+  const onToggleFirst = () => toggleFirst(!first);
+  const onToggleSecond = () => toggleSecond(!second);
+
+  return (
+    <>
```

Comment:
Very good contribution! :rocket:

This line will break in codesandbox, how about adding a `React.Fragment` or a `div` like you did in **Button.Default**?

https://gympass.github.io/yoga/components/button/default#default

Reply:
Hey everyone, studying this problem and from other codes written in CodeSandBox, this problem can be solved throughout Yoga by placing "react-scripts" as a dependency, as in the image below:

![image](https://user-images.githubusercontent.com/59899974/120684641-528b2300-c475-11eb-8d57-34a907277b00.png)

And it will look like that in CodeSandBox: 

![image](https://user-images.githubusercontent.com/59899974/120684868-9716be80-c475-11eb-887e-36e888c04e71.png)

I would like to hear from the team what they think about the implementation of this new dependency.
If it is consistent, I will open a PR with the correction. 🚀🚀

Reply:
Link to [pull request](https://github.com/Gympass/yoga/pull/263) for possible solution.